examples/flutter_gallery/lib/gallery/item.dart

```diff
@@ -306,6 +306,13 @@ List<GalleryItem> _buildGalleryItems() {
       routeName: CupertinoNavigationDemo.routeName,
       buildRoute: (BuildContext context) => new CupertinoNavigationDemo(),
     ),
+    new GalleryItem(
+      title: 'Pickers',
+      subtitle: 'Cupertino styled pickers',
+      category: 'Cupertino Components',
+      routeName: CupertinoPickerDemo.routeName,
+      buildRoute: (BuildContext context) => new CupertinoPickerDemo(),
+    ),
```

Comment:
see the comment at the top of this list.

Reply:
Thanks, done